content/organizations/managing-teams/managing-team-access-to-organization-packages.mdx

```diff
@@ -7,7 +7,7 @@ redirect_from:
 
 import shared from '~/shared.js'
 
-As an organization owner or team admin, you can add or remove package access to or from teams in your organization.
```

Comment:
I believe all mentions of "team admin" in this doc should be removed. See line 50, for example.